proto/substrait/algebra.proto

```diff
@@ -726,6 +730,47 @@ message NestedLoopJoinRel {
   substrait.extensions.AdvancedExtension advanced_extension = 10;
 }
 
+message DelimGetRel{
+   RelCommon common = 1;
```

Comment:
What is the input for DelimGetRel?  Presumably it's the same as the input for the closest DelimJoinRel either as a duplicate tree or a ReferenceRel to the same (I'm not aware of engine that has support for that yet though).  An input relation is likely needed to express that.

Reply:
The `DelimGetRel` has the duplicate eliminated columns from the closest `DelimJoinRel` as input.

I guess we could add a  use a ReferenceRel instead of explicitly having the column types here.